ote/src/cljs/ote/views/parking.cljs

```diff
@@ -1,51 +1,195 @@
+                    :show-option-short (tr-key [:enums ::t-service/day :short])}
+                   {:name  ::t-service/all-day
+                    :width "10%"
+                    :type  :checkbox
+                    :write (fn [data all-day?]
+                             (merge data
+                                    {::t-service/all-day all-day?}
+                                    (when all-day?
+                                      {::t-service/from (time/->Time 0 0 nil)
+                                       ::t-service/to   (time/->Time 24 0 nil)})))}
+
+                   {:name         ::t-service/from
+                    :width        "25%"
+                    :type         :time
+                    :cancel-label (tr [:buttons :cancel])
+                    :ok-label     (tr [:buttons :save])
+                    :write        (write ::t-service/from)
+                    :default-time {:hours "08" :minutes "00"}}
+                   {:name         ::t-service/to
+                    :width        "25%"
+                    :type         :time
+                    :cancel-label (tr [:buttons :cancel])
+                    :ok-label     (tr [:buttons :save])
+                    :write        (write ::t-service/to)
+                    :default-time {:hours "19" :minutes "00"}}]
+       :delete?   true
+       :add-label (tr [:buttons :add-new-service-hour])}
+
+      {:name         ::t-service/service-exceptions
+       :type         :table
+       :table-fields [{:name  ::t-service/description
+                       :label (tr* :description)
+                       :type  :localized-text}
+                      {:name  ::t-service/from-date
+                       :type  :date-picker
+                       :label (tr* :from-date)}
+                      {:name  ::t-service/to-date
+                       :type  :date-picker
+                       :label (tr* :to-date)}]
+       :delete?      true
+       :add-label    (tr [:buttons :add-new-service-exception])}
+
+      ;;
+
+      {:name ::t-service/maximum-stay
+       :type :interval})))
+
+(defn capacities [e!]
+  (form/group
+    {:label   (tr [:parking-page :header-facilities-and-capacities])
+     :columns 3
+     :layout  :row}
+
+    {:name         ::t-service/parking-capacities
+     :type         :table
+     :table-fields [{:name        ::t-service/parking-facility
+                     :type        :selection
+                     :show-option (tr-key [:enums ::t-service/parking-facility])
+                     :options     t-service/parking-facilities}
+                    {:name ::t-service/capacity :type :number}]
+     :add-label    (tr [:buttons :add-new-parking-capacity])
+     :delete?      true}))
+
+(defn charging-points [e!]
+  (form/group
+    {:label   (tr [:parking-page :header-charging-points])
+     :columns 3
+     :layout  :row}
+
+    {:name    ::t-service/charging-points
+     :columns 3
+     :type   :localized-text}))
```

Comment:
Can you add rows for this element?
When group contains only one input element, we have too much white space in that group, so it can be larger.